src/styles/StudentStyles/index.less

```diff
@@ -159,6 +163,7 @@
     display: flex;
     flex-direction: row;
     margin-right: 0%;
+    justify-content: space-between;
 }
 
 .top-left h3 {
```

Comment:
are empty/commented out classes for later, or can they be removed?

Reply:
Fixed!